translations/zh_CN/index.po

```diff
@@ -31,114 +35,116 @@ msgid ""
 "managing, persisting, sharing and reproducing the complex workflows "
 "associated with modern computational science and all associated data (see "
 ":ref:`features<intro:about>`)."
-msgstr "一个开源 Python 基础设施，可帮助研究人员自动化、管理、持久化、共享和复制与现代计算科学和所有相关数据有关的复杂工作流（见 :ref:`features<intro:about>`）。"
+msgstr ""
+"一个开源 Python 基础设施，可帮助研究人员自动化、管理、持久化、共享和复制与现代计算科学和所有相关数据有关的复杂工作流（见 "
+":ref:`features<intro:about>`）。"
 
 #: ../../source/index.rst:28
 msgid "**aiida-core version:** |release|"
-msgstr ""
+msgstr "**aiida-core 版本：** |release|"
 
 #: ../../source/index.rst:35
 msgid "Getting Started"
 msgstr ""
 
 #: ../../source/index.rst:39
 msgid "AiiDA installation, configuration and troubleshooting."
-msgstr ""
+msgstr "AiiDA 安装、配置和故障排除。"
 
 #: ../../source/index.rst:43
 msgid "To the installation guides"
-msgstr ""
+msgstr "安装指南"
```

Comment:
```suggestion
msgstr "To the installation guides"
```

Reply:
Need to keep the same link button for the card, otherwise, the link is not working. Same for others.